extensions/interactions/ImageClickInput/directives/oppia-interactive-image-click-input.component.ts

```diff
@@ -0,0 +1,284 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the ImageClickInput interaction.
+ *
+ * IMPORTANT NOTE: The naming convention for customization args that are passed
+ * into the directive is: the name of the parameter, followed by 'With',
+ * followed by the name of the arg.
+ */
+
+import { Component, ElementRef, Input, OnDestroy, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { AppConstants } from 'app.constants';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { ImageClickAnswer } from 'interactions/answer-defs';
+import { ImageClickInputCustomizationArgs, ImageWithRegions, LabeledRegion } from 'interactions/customization-args-defs';
```

Comment:
```suggestion
import { 
  ImageClickInputCustomizationArgs,
  ImageWithRegions,
  LabeledRegion 
} from 'interactions/customization-args-defs';
```

Reply:
I don't really see the need for this... it breaks the flow of imports which is hurting readability. When some of them are in one line and the others are not, it's kind of jarring and breaking.
This feels more readable
![Screenshot 2021-04-24 at 12 00 14 AM](https://user-images.githubusercontent.com/23002114/115914811-0e792b80-a490-11eb-8bdb-e8cf1fdeced0.png)
than this
![Screenshot 2021-04-24 at 12 01 45 AM](https://user-images.githubusercontent.com/23002114/115914965-441e1480-a490-11eb-8f6d-3bef94d4e99c.png)

The second one just looks broken. WDYT?

Reply:
TBH, for me the first option is more unreadable since the length of the lines, I have already come across this on some PR and Nithesh mentioned that the first approach is the one that you prefer. I'm okay with that as long as it is documented and properly explained why it is done this way.

Reply:
Not a blocker for this PR btw.

Reply:
I will put a point for discussion, either in the project leads meeting or the core maintainers' meeting. Let's not do the first one because I prefer it that way. I would really want to know what other folks' opinions are and decide what's best for the org.

Reply:
Yep, sure, let's discuss it there.